yarn.lock

```diff
@@ -7520,7 +7520,7 @@ fs-extra@^7.0.0:
     jsonfile "^4.0.0"
     universalify "^0.1.0"
 
```

Comment:
- [ ] Large binaries should not be in git -- don't merge the 4MB stuff into `master` 

In general, we only keep code in git (I know there are a few existing violations) since these binaries can't be compressed.  Large resources usually go into things like `git-lfs`, and for our case, should be in the CDN.

I know we talked about this in Slack -- this is just to keep track.

As long as it hasn't reached the `master` chain, we can prune off the branch later and the periodic `git gc` at github should remove it from the pack.

Reply:
Yeah I've already asked Niko about the move to the CDN. He'll figure it out later today.